src/services/StudentService.js

```diff
@@ -0,0 +1,65 @@
+import { prisma } from '../PrismaClient.js';
+
+import { EntryExists } from '../errors/EntryExists.js';
+
+async function findAll() {
+  const students = await prisma.student.findMany();
+
+  return students;
+}
+
+async function add(data) {
+  try {
+    const student = await prisma.student.create({
+      data: data,
+    });
+    return student;
+  } catch (error) {
+    if (error.code == 'P2002') {
+      throw new EntryExists();
+    } else {
+      throw error;
+    }
+  }
+}
+
+async function read(query, type) {
```

Comment:
Evitar magic numbers.
```suggestion
async function read(query, byEmail) {
```